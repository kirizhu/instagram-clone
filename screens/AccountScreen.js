import { StyleSheet, Text, View } from "react-native";
import React from "react";

const AccountScreen = () => {
  return (
    <View style={styles.container}>
      <Text style={styles.text}>AccountScreen</Text>
    </View>
  );
};

export default AccountScreen;

const styles = StyleSheet.create({
  container: {
    backgroundColor: "black",
    justifyContent: "center",
    alignItems: "center",
  },
  text: {
    color: "white",
  },
});
