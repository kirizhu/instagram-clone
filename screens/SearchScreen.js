import { StyleSheet, Text, View } from "react-native";
import React from "react";

const SearchScreen = () => {
  return (
    <View style={styles.container}>
      <Text style={styles.text}>SearchScreen</Text>
    </View>
  );
};

export default SearchScreen;

const styles = StyleSheet.create({
  container: {
    backgroundColor: "black",
    justifyContent: "center",
    alignItems: "center",
  },
  text: {
    color: "white",
  },
});
