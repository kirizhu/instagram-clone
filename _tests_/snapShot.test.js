import * as React from "react";
import { create } from "react-test-renderer";
import AccountScreen from "../screens/AccountScreen";
import SearchScreen from "../screens/SearchScreen";
const AccountTree = create(<AccountScreen />);
const SearchTree = create(<SearchScreen />);
describe("Testing react components", () => {
  test("snapshot", () => {
    expect(AccountTree).toMatchSnapshot();
  });
  test("snapshot", () => {
    expect(SearchTree).toMatchSnapshot();
  });
});
