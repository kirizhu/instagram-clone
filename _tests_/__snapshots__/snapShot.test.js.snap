// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing react components snapshot 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "black",
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      Object {
        "color": "white",
      }
    }
  >
    AccountScreen
  </Text>
</View>
`;

exports[`Testing react components snapshot 2`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "black",
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      Object {
        "color": "white",
      }
    }
  >
    SearchScreen
  </Text>
</View>
`;
